assets/styles.css

```diff
@@ -0,0 +1,18 @@
+html {
+	background-color: grey;
+	background-image: url("/assets/bike-wall.png");
```

Comment:
I'd prefer not committing any binary files to the repository. In general, it isn't good practice. What is this image?

Reply:
It's the photo I'm using for the background wallpaper; I made sure that it was copyright labeled for reuse. I don't know why it is specifically being tagged as binary, but I switched it out for a web url instead and removed the local files.